GitUI/ScriptsEngine/ScriptsManager.ScriptRunner.cs

```diff
@@ -71,8 +71,8 @@ private static CommandStatus RunScriptInternal(ScriptInfo script, IWin32Window o
                     PowerShellHelper.RunPowerShell(command, argument, uiCommands.GitModule.WorkingDir, script.RunInBackground);
 
                     // 'RunPowerShell' always runs the script detached (yet).
-                    // Hence currently, it does not make sense to set 'needsGridRefresh' to '!scriptInfo.RunInBackground'.
-                    return new CommandStatus(executed: true, needsGridRefresh: false);
+                    // Hence currently, it does not make sense to call uiCommands.RepoViewModel.RefreshRevisions() if '!scriptInfo.RunInBackground'.
```

Comment:
```suggestion
                    // Hence currently, it does not make sense to call 'uiCommands.RepoViewModel.RefreshRevisions()' if '!scriptInfo.RunInBackground'.
```
The ultimate nitpick